src/models/test/StyleSheet.test.js

```diff
@@ -65,6 +65,10 @@ describe('styleSheet', () => {
       const css = styleSheet.getCSS({ min: false });
       expect(stripWhitespaceExceptNewline(css)).toEqual('.sc-a {}.c {width: 100%;}\n.sc-b {}.d {text-decoration: none;}')
     })
+
+    it('should not throw an error if no styled-components are rendered', () => {
+      const css = styleSheet.getCSS();
```

Comment:
This is a good start, but this would be even better:

```JS
expect(() => styleSheet.getCSS()).toNotThrow()
```

To make it very clear what this should do 😊